app/pathogen/sarscov2/dashboard/(map)/SarsCov2Map.tsx

```diff
@@ -0,0 +1,99 @@
+import { MapShadingLegend } from "@/app/pathogen/arbovirus/dashboard/(map)/MapShadingLegend";
+import { MapEstimateSummary } from "@/components/ui/pathogen-map/map-estimate-summary";
+import { MapSymbology } from "./map-config";
+
+export function SarsCov2Map() {
+  const state = useContext(SarsCov2Context);
+  const { data } = useNewSarsCov2Data();
+
+  if (!data) {
+    return <span> Loading... </span>;
+  }
+
+  return (
+    <>
+      <div className={"w-full h-full p-0"}>
+        <PathogenMap
+          id="sarsCov2Map"
+          baseCursor=""
+          sourceId="sc2-[GENERATED-SOURCE-ID]"
+          layers={[
+            {
+              id: "SARS-CoV2-pins",
+              type: "circle",
+              isDataUsedForCountryHighlighting: true,
+              cursor: "pointer",
+              filter: ["!", ["has", "point_count"]],
+              layerPaint: {
+                "circle-color": [
+                  "match",
+                  ["get", "scope"],
+                  "National",
+                  MapSymbology.StudyFeature.National.Color,
+                  "Regional",
+                  MapSymbology.StudyFeature.Regional.Color,
+                  "Local",
+                  MapSymbology.StudyFeature.Local.Color,
+                  "Sublocal",
+                  MapSymbology.StudyFeature.Sublocal.Color,
+                  MapSymbology.StudyFeature.Default.Color,
+                ],
+                "circle-radius": [
+                  "match",
+                  ["get", "scope"],
+                  "National",
+                  MapSymbology.StudyFeature.National.Size,
+                  "Regional",
+                  MapSymbology.StudyFeature.Regional.Size,
+                  "Local",
+                  MapSymbology.StudyFeature.Local.Size,
+                  "Sublocal",
+                  MapSymbology.StudyFeature.Sublocal.Size,
+                  MapSymbology.StudyFeature.Default.Size,
+                ],
+                "circle-stroke-color": [
+                  "case",
+                  ["boolean", ["feature-state", "isSelected"], false],
+                  "black",
+                  "white",
+                ],
+                "circle-stroke-width": 3,
+                "circle-stroke-opacity": [
+                  "case",
+                  ["boolean", ["feature-state", "hover"], false],
+                  1,
+                  [
+                    "case",
+                    ["boolean", ["feature-state", "isSelected"], false],
+                    1,
+                    0,
+                  ],
+                ],
+                "circle-opacity": [
+                  "case",
+                  ["boolean", ["feature-state", "isBlurred"], false],
+                  0.2,
+                  0.6,
+                ],
+              },
+            },
+          ]}
+          clusteringSettings={{
+            clusteringEnabled: false
```

Comment:
I don't know if we want to do some clustering later on or something but turning it off makes my life a lot easier for the time being so that's what I did hahaha